src/customlinecap.h

```diff
@@ -14,6 +14,7 @@
 GpStatus WINGDIPAPI GdipCreateCustomLineCap (GpPath *fillPath, GpPath *strokePath, GpLineCap baseCap, float baseInset, GpCustomLineCap **customCap);
 GpStatus WINGDIPAPI GdipDeleteCustomLineCap (GpCustomLineCap *customCap);
 GpStatus WINGDIPAPI GdipCloneCustomLineCap (GpCustomLineCap *customCap, GpCustomLineCap **clonedCap);
+GpStatus WINGDIPAPI GdipGetCustomLineCapType( GpCustomLineCap* customCap, CustomLineCapType* capType);
```

Comment:
nit: space before the `(`, not after :)

Reply:
also `*` should be stuck to the argument, not the type, like everywhere else: `GpCustomLineCap *customCap`